src/utils/flatten.js

```diff
@@ -58,6 +59,10 @@ const flatten = (
         : ruleSet.concat(chunk)
     }
 
+    if (chunk instanceof Keyframes) {
+      return ruleSet.concat(chunk)
```

Comment:
An explicit string instead of coercion has a better chance of not being deoptimised

Reply:
It’d actually be more interesting to pass on the current StyleSheet I to flatten from ComponentStyle, evaluate the keyframes, inject it, and return its name :)

Reply:
Essentially less efficient and introduces side effects but it can be sharped out as well